### DISCUSSION TITLE
Font Size of Button text

### DISCUSSION THREAD

**Discussion by SilverPirateX** [TomSchimansky/CustomTkinter#104] [Q&A]
is there is any way i can change the font size of CTkButton text ?
I usually use #font=("Times", "30", "bold") for example in the standard tkinter but when used for CTkButton somehow it says that it doesn't have such option.
![Screenshot at 2022-05-23 12-39-44](https://user-images.githubusercontent.com/53333848/169800855-1f06c0dc-ab86-4d04-902b-a3ad697a66a3.png)

**Comment by IshanJ25** [comment]
```python
# this is a python comment
# it is not executed by compiler
```

**Comment by KAILO776** [comment]
for CTK objects, you can use the ,text_font attribute, so to use your above example 
```
custom_font =("Times",30,'bold')
btn = customtkinter.CTkButton(master=root, text="something", text_font=custom_font , command=func())
btn .pack()
```
